scripts/AI/ai_fight.gd

```diff
@@ -51,6 +57,27 @@ func _process(delta):
 				break
 		return # find player, just come back next frame with a valid player instance	
 	dist = thrall.global_position.distance_to(player.global_position)
+	# Handle patrol logic of skeleton 01
+	if thrall.name == "skele_01" and dist > dist_see and not (dist <= dist_attack or dist <= dist_see):
```

Comment:
If we make the patrol destination variable public, we could make this specifically refer to the entity that runs this script. Then we could even make a visual node in godot so that future developers can click where they want NPC's to patrol.